### DISCUSSION TITLE
Some suggestions about Apollo

### DISCUSSION THREAD

**Discussion by ZhaoZelin2000** [ClassicOldSong/Apollo#49] [Ideas]
The current Apollo can open multiple instances for multiple displays. However, the operation process is still quite complex. Is the author interested in developing Moonlight or Artemis clients to select how many instances to open and adjust the display resolution and refresh rate of each instance. In addition, if it is possible to share clipboard and copy files, I believe Apollo would be the most powerful software suitable for gaming and office use

**Comment by ClassicOldSong** [comment]
All these features you have mentioned are already planned. Clipboard sync will be implemented first, then multi display capture.
